### PR TITLE
Updated flutter/examples to further conform to new embedding: removed references to FlutterApplication, deleted all MainActivity's that were not necessary, removed all direct invocations of GeneratedPluginRegistrant. (#22529)

### PR DESCRIPTION
**Pull request by matthew-carroll**
Updated flutter/examples to further conform to new embedding: removed references to FlutterApplication, deleted all MainActivity's that were not necessary, removed all direct invocations of GeneratedPluginRegistrant. (#22529)

### CODE REVIEW

File: examples/flutter_gallery/android/app/src/main/java/io/flutter/demo/gallery/MainActivity.java [hunk 1]
```diff
@@ -16,10 +14,4 @@
     public FlutterGalleryInstrumentation getInstrumentation() {
         return instrumentation;
     }
-
-    @Override
-    public void configureFlutterEngine(@NonNull FlutterEngine flutterEngine) {
-        GeneratedPluginRegistrant.registerWith(flutterEngine);
-        instrumentation = new FlutterGalleryInstrumentation(flutterEngine.getDartExecutor());
```
Comment (xster) [inline examples/flutter_gallery/android/app/src/main/java/io/flutter/demo/gallery/MainActivity.java]: I think this line is still needed

File: examples/flutter_view/android/app/src/main/java/com/example/view/MainActivity.java [hunk 2]
```diff
@@ -7,16 +7,19 @@
 import androidx.appcompat.app.ActionBar;
```
Comment (xster) [inline examples/flutter_view/android/app/src/main/java/com/example/view/MainActivity.java]: Cool, thanks for catching this one

**Comment by xster** [comment]
LGTM, thanks!

File: examples/flutter_gallery/android/app/src/main/java/io/flutter/demo/gallery/MainActivity.java (hunk 1 above)
Comment (matthew-carroll) [reply examples/flutter_gallery/android/app/src/main/java/io/flutter/demo/gallery/MainActivity.java]: Thanks. I missed that line. Restored.

**Pull request merged**